src/scripts/components/header/header.styl

```diff
@@ -24,7 +24,7 @@
   white-space: nowrap
   font-size: emCalc(18px)
 
-.playButton
+.controls
```

Comment:
Sorry but please use a more general name like `rightContent` or something. This could be **anything** not only controls